src/Microsoft.AspNetCore.Razor.Runtime/project.json

```diff
@@ -1,5 +1,10 @@
 {
-  "description": "Runtime components for rendering Razor pages.",
+  "description": "Runtime components for rendering Razor pages and implementing tag helpers.\r\nCommonly used types:\r\nMicrosoft.AspNetCore.Razor.TagHelpers.HtmlAttributeNameAttribute\r\nMicrosoft.AspNetCore.Razor.TagHelpers.HtmlTargetElementAttribute\r\nMicrosoft.AspNetCore.Razor.TagHelpers.ITagHelper\r\nMicrosoft.AspNetCore.Razor.TagHelpers.TagHelper",
```

Comment:
Side question: Do these Windows line endings work when the package is built on Linux / OSx?